Dockerfile

```diff
@@ -2,13 +2,15 @@ FROM node:6.10.3-alpine
 
 WORKDIR /app
 
-COPY . .
+COPY package.json .
 
 RUN apk add --no-cache --virtual .build-dependencies git && \
     sed -i -e 's/"mongojs": "0.18.2"/"mongojs": "2.4.0"/g' package.json && \
     yarn install && \
     apk del .build-dependencies
 
+COPY . .
+
```

Comment:
`docker build` [creates an AUFS layer for each line](https://medium.com/@jessgreb01/digging-into-docker-layers-c22f948ed612) in the `Dockerfile`.
When you run subsequent `docker build`s, only the layers that change are built. Therefore, if you haven't changed `package.json` (in my change), it won't rebuild the `COPY package.json .` or `RUN ...` layer.
In the `Dockerfile` before that change, it would _always_ build the `RUN ...` layer if _any_ code had changed.
This simple change brings the build time of a subsequent seagull `docker build` from 45s to 2s on my machine.